README.md

```diff
@@ -1,26 +1,266 @@
+- **RawDataFFT.json** - Contains continuous information streamed from the onboard (on-chip) FFT engine. The structure and timing information is similar to the time domain files.
+- **RawDataPower.json** - Contains continuous information streamed from the on board FFT engine in select power bands. The data rate is set by the FFT engine, and can be very fast (1ms) and very slow (upper limit is in the hours or days range). This is the raw input in the onboard embedded adaptive detector. The raw power is much less data intensive than the FFT data. You can stream up to 8 power domain channels (2/each TD channel) simultaneously. Note that the actual bandpass limits are not contained in RawDataPower.json but rather in DeviceSettings.json. If these values are changed during a recording, mapping will be required from the times in DeviceSettings to the data in RawDataPower.
+- **AdaptiveLog.json** - Contains any information from the embedded adaptive detector. The structure and timing information is similar to the time domain files.
+- **StimLog.json** - Contains information about the stimulation setup (e.g. which group, program, rate and amplitude the device is currently using for stimulation). The structure and timing information is similar to the time domain files. Much of this information is duplicated in DeviceSettings.json.
+- **ErrorLog.json**- Contains information about errors. Not currently used.
+- **EventLog.json** - Contains discrete information we write into the device. These can be experimental timings or patient report of his state if streaming at home. Note that this information only contains timing information in computer time, whereas all other .json files have timing relative to (on-board) INS time. [See section below on timestamp and systemTick](https://github.com/openmind-consortium/Analysis-rcs-data/tree/DocumentationUpdate#systemtick-and-timestamp)
+- **DiagnosticsLog.json** - Contains discrete information that can be used for error checking.
+- **TimeSync.json**: Not currently used
+
+Note that in each recording session, all .json files will be created and saved. If a particular datastream (e.g. FFT) is not enabled to stream, that .json file will be mostly empty, containing only minimal metadata.
+
+### Data imported into Matlab 
+
+- **RawDataTD.json** --> timeDomainData:
+  - key0: Channel 0 on the first INS bore (assuming no bridging); contains numerical data in millivolts
+  - key1: Channel 1 on the first INS bore (assuming no bridging); contains numerical data in millivolts
+  - key2: Channel 2 on the first INS bore (assuming no bridging); contains numerical data in millivolts
```

Comment:
second INS bore?

same question for key3

Reply:
Updated to remove mention of different bores